### PR TITLE
Initial working Vagrantfile plus ansible for Mac

### PR DESCRIPTION
**Pull request by sauloperez**
Turns out that with `xenial64` there's no need to hack the ubuntu password at all

### CODE REVIEW

File: development.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@
 - hosts: katuma-dev
   remote_user: ubuntu
   sudo: yes
+  # TODO: Doc in wiki. A manos entrando a la maquina. No hay mas. Inline script con vagrant
+  # pre_tasks:
+    # - name: apt-get update 
+      # raw: apt-get update -qq 
+    # - name: Install python 2.7 
+      # raw: apt-get install -qq python2.7  # Missing sudo? doesn't it run with sudo?
```
Comment (enricostano) [inline development.yml]: docs++

:muscle:

Comment (sauloperez) [reply development.yml]: Might be worth mentioning it in the docs, but this already handled by the inline provisioning in the vagrant file

**Comment by enricostano** [comment]
Same `npm install` error here:

```
npm ERR! Linux 4.4.0-45-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "i"
npm ERR! node v4.6.1
npm ERR! npm  v2.15.9
npm ERR! path npm-debug.log.3119109857
npm ERR! code EROFS
npm ERR! errno -30
npm ERR! syscall open

npm ERR! rofs EROFS: read-only file system, open 'npm-debug.log.3119109857'
npm ERR! rofs This is most likely not a problem with npm itself
npm ERR! rofs and is related to the file system being read-only.
npm ERR! rofs 
npm ERR! rofs Often virtualized file systems, or other file systems
npm ERR! rofs that don't support symlinks, give this error.
```

Will try to `npm install` a dummy project with only few dependencies to see if it's some specific package failing.

**Comment by sauloperez** [comment]
I'm not sure this only related to node. As I found out the kernel sets the fs to read-only when I/O fail, possible due to a timeout. Then, there's no way to enable write access. I'll share the dmesg output that I got.

I'd try to understand what's the first failing operation and reproduce it without npm. If we can, we know it's something about the VMs fs. If not, it definitely is npm.

**Comment by enricostano** [comment]
LOL using yarn works!
https://yarnpkg.com/en/docs/install
`f***ing hipsters`

EDIT: I also created a dummy node https://gist.github.com/enricostano/d1f06613bfaa4e7e9188a6ae7744e55d

and it's broken with npm 2, but works with npm 3.........

**Comment by andresgutgon** [comment]
image

**Comment by sauloperez** [comment]
Wtf, so it turns out to be due to npm 2??? What do they do???

**Comment by enricostano** [comment]
Rebase from `master` and try again please :pray:

**Comment by enricostano** [comment]
Rebase from master to get the git submodules fix please :grimacing:

**Pull request merged**